build.gradle

```diff
@@ -449,6 +449,7 @@ project(':iceberg-orc') {
   dependencies {
     implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
     api project(':iceberg-api')
+    implementation project(':iceberg-common')
```

Comment:
Are we sure we need to add `iceberg-common`? Looking at the definition of `iceberg-common`, it appears to be the same as line 450's `iceberg-bundled-guava`, using shadow configuration.

https://github.com/apache/iceberg/blob/718ff6ae514d7a16077c21e5281e765ba5fa120b/build.gradle#L199-L203

I do see it duplicated multiple times though in other places.